ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestsecurity.py

```diff
@@ -36,3 +36,45 @@ def test_definition(self) -> dict | None:
         anta_tests.append({"VerifyAPIHttpsSSL": {"profile": profile, "result_overwrite": {"custom_field": f"eAPI HTTPS SSL Profile: {profile}"}}})
 
         return {self.anta_module: anta_tests}
+
+
+class AvdTestIPSecurity(AvdTestBase):
+    """
+    AvdTestIPSecurity class for IP security connection tests.
```

Comment:
I know this is following the other test classes docstring, but I think for WAN devices related tests we should give more details (following the idea of the other comments).